test/fit_tests/README.md

```diff
@@ -0,0 +1,161 @@
+                          'extended', default: 'all'
+      -stack STACK        stack label (test bed), overrides -ora
+      -ora ORA            OnRack/RackHD appliance IP address or hostname, default:
+                          localhost
+      -version VERSION    OnRack package install version, example:onrack-
+                          release-0.3.0, default: onrack-devel
+      -template TEMPLATE  path or URL link to OVA template or OnRack OVA, default
+                          from global_config.json
+      -xunit              generates xUnit XML report files
+      -list               generates test list only
+      -sku SKU            node SKU, example:Phoenix, default=all
+      -obmmac OBMMAC      node OBM MAC address, example:00:1e:67:b1:d5:64
+      -nodeid NODEID      node identifier string of a discovered node, example:
+                          56ddcf9a8eff16614e79ec74
+      -http               forces the tests to utilize the http API protocol
+      -https              forces the tests to utilize the https API protocol
+      -port PORT          API port number override, default from
+                          global_config.json
+      -v V                Verbosity level of console output, default=0, Built Ins:
+                          0: No debug, 2: User script output, 4: rest calls and
+                          status info, 6: other common calls (ipmi, ssh), 9: all
+                          the rest
+
+This example will run the RackHD installer onto stack 1 via the wrapper script:
+
+    python run_tests.py -stack 1 -test deploy/run_rackhd_installer.py
+
+
+The -stack or -ora argument can be omitted when running on the server or appliance. The test defaults to localhost:8080 for API calls.
+
+This example will run the smoke test from the appliance node or the default Vagrant test bed:
+
+    python run_tests.py -test tests -group smoke
+
+
+Alternatively tests can be run directly from nose. Runtime parameters such as ORA address must be set in the environment.
+
+The following example will run all the entire test harness from a third party machine to ORA at 192.168.1.1:
+
+    export ORA=192.168.1.1
+    nosetests -s tests
+
+
+## Running individual tests
+
+Individual test scripts or tests may be executed using the following 'Nose' addressing scheme:
+
+    test_script_path:classname.testname
+
+
+For example, to run the test 'test_rackhd11_api_catalogs' in script 'tests/rackhd11/test_rackhd11_api_catalogs.py' on stack 1:
+
+    python run_tests.py -stack 11 -test tests/rackhd11/test_rackhd11_api_catalogs.py:rackhd11_api_catalogs.test_api_11_catalogs
+
+
+## Running FIT tests on Vagrant RackHD
+
+Install the Vagrant box using the instructions here:
+https://github.com/RackHD/RackHD/tree/master/example
+
+Install the Quanta simulator:
+
+    vagrant up quanta_d51
+
+Log into the Vagrant box:
+
+    vagrant ssh dev
+
+Clone FIT tests:
+
+    git clone https://github.com/onrack2/fit_tests
```

Comment:
we wouldn't want to keep this in place once this PR is merged, correct? The test code will already be at `~/src/test` on the vagrant instance, perhaps use that instead?